### DISCUSSION TITLE
How to have copilot

### DISCUSSION THREAD

**Discussion by philip121202** [community/community#121321] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

wanted to register on github copilot but i was asked to upgrade my two-factors authentification and i made it but they were two other thing to do that i did'nt read .so what should i do to fix the remaining error

**Comment by haululazkiyaa** [comment]
You can have copilot for free if you are student and verify your identity first in github education.

**Comment by itzzjb** [comment, marked as answer]
If you are a student, you can get the GitHub student pack. It might take 3-5 days to verify that you are a student using your documents. Then you will be able to use Copilot, Github Actions etc fro free. Otherwise you need to pay for them.
